styles.css

```diff
@@ -90,3 +90,224 @@ ul {
   padding-right: 10px;
   align-items: center;
 }
+
+.works {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  padding: 30px 20px;
+}
+
+.recent-works {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 40px;
+  margin-top: 70px;
+  color: #091e42;
+  line-height: 52px;
+}
+
+hr {
+  color: black;
+  background-color: black;
+  width: 327px;
+  height: 0;
+  margin-bottom: 60px;
+}
+
+.bgimage {
+  background-image: url(images/works.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  border: inset 170px transparent;
+  box-sizing: border-box;
```

Comment:
- [ OPTIONAL]  Good job adding the section requested by the previous reviewer :+1: , Kindly stick to the design as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using), regarding the background size and the font :recycle: 

  - Your design:
![yourdesign](https://user-images.githubusercontent.com/97834160/205331059-f72ef074-40d7-48bb-879c-5a51284dbd2e.png)

  - Figma design:
![figmad](https://user-images.githubusercontent.com/97834160/205331028-eaf257f3-5cbd-4795-b545-cde835daf8e6.png)